### PR TITLE
icon package technotes

### PR DESCRIPTION
**Pull request by mitchelloharawild**
https://github.com/ropenscilabs/icon/issues/14

### CODE REVIEW

**Comment by maelle** [comment]
Looks good to me, thanks a lot and very exciting development plans! cc @stefaniebutland

File: content/technotes/2018-05-10-icon.Rmd [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+authors:
+  - name: Mitchell O'Hara-Wild
+    url: https://github.com/mitchelloharawild
+date: 2018-05-10
+output: 
+categories:
+- technotes
+topicid: 
+tags:
+- R
```
Comment (stefaniebutland) [inline content/technotes/2018-05-10-icon.Rmd]: please add tags:

`- unconf`
`- ozunconf`

File: content/technotes/2018-05-10-icon.Rmd [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+slug: "icon"
+title: "icon: web icons for rmarkdown"
+authors:
+  - name: Mitchell O'Hara-Wild
+    url: https://github.com/mitchelloharawild
+date: 2018-05-10
```
Comment (stefaniebutland) [inline content/technotes/2018-05-10-icon.Rmd]: change date to 2018-05-15 in YAML and filename and I can publish

File: content/technotes/2018-05-10-icon.Rmd [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+The package also provides a second interface which allows for tab completion of font names. To include the rocket icon above, you could instead use `` `r '\x60r icon::fa_rocket()\x60'` ``. This is a useful interface for if you are unfamiliar with the available icons or their names.
+
+Each icon function also provides support for modifying the attributes of the icon such as size, colour, rotation, and animation. We can make the rocket icon spin using `` `r '\x60r icon::fa_rocket(animate = "spin")\x60'` ``.
+
+# R in icons
+The long-awaited [R Project icon](http://cran.r-project.org/logo.html) has been added to the Font Awesome library in version 5.0.11. This addition comes nearly four years since the icon's [first request](https://github.com/FortAwesome/Font-Awesome/issues/3862) after being brought into the spotlight of [#rstats](https://twitter.com/hashtag/rstats) with this [tweet](https://twitter.com/LockeData/status/987286302721806337).
```
Comment (stefaniebutland) [inline content/technotes/2018-05-10-icon.Rmd]: Optional: you could add "with this tweet by [LockeData](https://twitter.com/LockeData)." to give more of a direct shoutout / credit for the signal boost.

Comment (mitchelloharawild) [reply content/technotes/2018-05-10-icon.Rmd]: Is it possible to embed tweets in posts?
https://dev.twitter.com/web/embedded-tweets

Comment (stefaniebutland) [reply content/technotes/2018-05-10-icon.Rmd]: Yes! embedding would be great

**Comment by stefaniebutland** [comment]
The preview looks good to go. I've asked for help to check on the source of the failing checks.

**Pull request merged**